test/modules/Modules.Test/TestResultCoordinator/Reports/DirectMethod/ReportDataForOneSource.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Modules.Test.TestResultCoordinator.Reports.DirectMethod
```

Comment:
This is used for test only, right? you should move it to Modules.Test project.

Reply:
Rename it to more specific, e.g. DirectMethodReportDataWithSenderSouceOnly and DirectMethodReportDataWithSenderAndReceiverSources.

Reply:
This is already in Modules.Test project. What do you mean?